### PR TITLE
sdl2_net: update to 2.2.0

### PR DESCRIPTION
**Pull request by neheb**
ping @Jan200101

### CODE REVIEW

File: subprojects/packagefiles/sdl2_net/meson.build [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
+)
 
 sdl2_net_dep = declare_dependency(
-    link_with : [sdl2_net],
-    dependencies: plat_libs,
-    version : meson.project_version(),
-    include_directories : '.')
+  link_with: sdl2_net,
+  dependencies: plat_libs,
+  include_directories: '.',
```
Comment (xclaesse) [inline subprojects/packagefiles/sdl2_net/meson.build:38]: you never need `'.'`

**Comment by xclaesse** [comment]
Other than small nitpicking, LGTM

File: subprojects/packagefiles/sdl2_net/meson.build (hunk 1 above)
Comment (neheb) [reply subprojects/packagefiles/sdl2_net/meson.build:38]: it is for declare_dependency()

Comment (xclaesse) [reply subprojects/packagefiles/sdl2_net/meson.build:38]: Ah right, indeed.

**Pull request merged**